docs/refs.bib

```diff
@@ -38,3 +38,53 @@ @inproceedings{hardt2016equality
   year      = {2016},
   url       = {https://proceedings.neurips.cc/paper/2016/hash/9d2682367c3935defcb1f9e247a97c0d-Abstract.html}
 }
+
+@inproceedings{10.1145/3287560.3287598,
+author = {Selbst, Andrew D. and Boyd, Danah and Friedler, Sorelle A. and Venkatasubramanian, Suresh and Vertesi, Janet},
```

Comment:
danah boyd prefers to be spelled in lowercase. That doesn't come through in the paper as the authors are shown in a font that has only caps.

Reply:
I'll fix that.